### ISSUE TITLE
Chapter 5, Ex. 9

### ISSUE THREAD

**Issue by jens321** [ilmoi/MMLBook#8]
Hi, 

For problem #9 in chapter 5, I think there is currently a missing partial with respect to **v** when computing dq/d**v**. Essentially, I believe this should be 

dq/d**v** = aq/a**z** * a**z**/a**v** + a**q**/a**v** * d**v**/d**v** = aq/a**z** * a**z**/a**v** + a**q**/a**v**

the solution currently does not include this last term a**q**/a**v**. (Note that "a" was used throughout to denote partial derivative.)

**Comment by NebiyouTen** [comment]
I think you are comparing two different approaches. I believe you are expecting, `ag/az` to be equal  to                  `p'(x, t(e,v)) / p(x, t(e,v))` - `q'(t(e,v),v) / q(t(e,v), v)` (which is the `t'(e,v)` term factored out from the equation in the solution manual). but note these are derivatives wrt `v` not `z`, which is different from partial derviatve wrt `z` where `t` is a constant. Hence, you will have an intermidate `aq/az` and `ap/az` terms.

**Comment by AlbertoGuastalla** [comment]
Yes, I also think that the author has forgotten the aq/av term...
Since the function q depends on two variables (z, v) and not only on z.
